.github/workflows/migration-labeler.yml

```diff
@@ -0,0 +1,66 @@
+name: Issue Migration Labeler
+
```

Comment:
Could you add a comment like `# owner: @vheinila` here to keep consistency in us documenting who should be contacted for questions regarding a workflow please?